src/scvi/nn/_base_components.py

```diff
@@ -96,7 +96,9 @@ def __init__(
                             nn.Dropout(p=dropout_rate) if dropout_rate > 0 else None,
                         ),
                     )
-                    for i, (n_in, n_out) in enumerate(zip(layers_dim[:-1], layers_dim[1:]))
+                    for i, (n_in, n_out) in enumerate(
+                        zip(layers_dim[:-1], layers_dim[1:], strict=False)
```

Comment:
again

Reply:
Apparently strict=False required for base_component, otherwise tests not pass